go.mod

```diff
@@ -156,17 +155,8 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
 
 // TODO: remove when middleware will be implemented
-replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.2.1-hooks
+replace github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.0.0-hooks-2
 
-replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
+replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.2
```

Comment:
v0.17.2 had some race condition / concurrent access issues -- not sure if this can manifest in Ethermint / Web3 APIs: https://github.com/cosmos/iavl/compare/v0.17.2...v0.17.3 but perhaps better to use the latest one?

Reply:
updated